scripts/loadAssets.py

```diff
@@ -2,116 +2,119 @@
-        if asset_exists:
-            if file in always_update_whitelist:
-                logger.info("File \""+str(file)+"\" should always be updated.")
-                update_existing_document(es_index, es_type, es_id, full_file_path)
-            else:
-                es_outdated, version_of_disk_file, version_of_es_file = es_version_is_outdated(es_index,
-                                                                                             es_type,
-                                                                                             es_id,
-                                                                                             full_file_path)
-                if es_outdated:
-                    logger.info("File \"" + str(file) + "\" is outdated and requires update from version " + str(version_of_es_file) +
-                             " to version " + str(version_of_disk_file) + ". Updating it now...")
-                    update_existing_document(es_index, es_type, es_id, full_file_path)
-                else:
-                    logger.info("Current version of file \"" + str(file) + "\" in Elasticsearch is up-to-date. Version: " + str(version_of_es_file))
-        else:
-            logger.info("File \"" + str(file) + "\" doesn't exist in Elasticsearch. Creating it now...")
-            created, created_ret = create_document_from_file(es_index, es_type, es_id, full_file_path)
-            logger.info("Create returns: " + created_ret)
+        created, created_ret = put_document_from_file(es_id, full_file_path)
+        if not created:
+            logger.warning("Failed to load kibana object: " + file)
 
 
 # ----------------- MAIN -----------------
 def main():
 
-    # Create arrays of the filenames in each of the resource dirs
-    dashboards = [filename for filename in listdir(dashboards_path) if isfile(join(dashboards_path, filename))]
-    visualizations = [filename for filename in listdir(visualizations_path) if isfile(join(visualizations_path, filename))]
-    searches = [filename for filename in listdir(searches_path) if isfile(join(searches_path, filename))]
-
-    # Load all the artifacts appropriately
-    logger.info("================================== DASHBOARDS ==================================")
-    load_assets(esUtil.KIBANA_INDEX, dashboard_type, dashboards_path, dashboards)
-    logger.info("================================== VISUALIZATIONS ==================================")
-    load_assets(esUtil.KIBANA_INDEX, visualization_type, visualizations_path, visualizations)
-    logger.info("================================== SEARCHES ==================================")
-    load_assets(esUtil.KIBANA_INDEX, search_type, searches_path, searches)
+    es_ok = check_elasticsearch_health()
+    if es_ok:
+        # Create arrays of the filenames in the resources directory
+        objects = [filename for filename in listdir(resources) if isfile(join(resources, filename))]
```

Comment:
this iteration could get moved into the `load_assets` function, with the only parameter to `load_assets` being the `path_to_files`